docs/1-Applying/resumes.md

```diff
@@ -78,24 +64,50 @@ Volunteer experience
 
 Small section for your other interests
 
-- I like to put that I sing in choirs and involved with certain clubs - it's important to me to have a life outside of CS-y things, and many employers appreicate this too. Some will talk about them a bit during interviews to break the ice!
+- I like to put that I sing in choirs and involved with certain clubs - it's important to me to have a life outside of CS-y things, and many employers appreciate this too. Some will talk about them a bit during interviews to break the ice!
 
 **Note**: You don't need all these sections and don't have to do them the way I advise!! Think about what skills you have that you can bring to this job, and find the things in your life that most prove you're qualified and figure out the best way to organize those things on your resume. Everyone is different, so everyone has a slightly different resume that will advertise them best.
 
 I'd recommend checking out other people's resumes too, as some of them have sections in their resumes that I don't personally use - these are just the sections that I like and work best for me.
 
-### How to talk about things
+### Show don't tell
 
-- use action verbs - they help describe the skills you've used, and starting bullet points with them helps keep your descriptions short and powerful - Google "action verbs" to find a bunch of lists of words to use
-- add dates when possible - for work experience, projects, volunteering, etc.
-- bullet points should be concise and easy to scan, so ideally only one line long and rarely longer than two lines
-- if you have more than 4 bullet points for one section, it'll become a big block of text and will become harder to scan as well
+As much as possible, don't list your skills without context proving you have them. Anyone can write "good communicator" on their resume, and unless you talk about something you did that needed very good communication skills, that line is essentially useless. In general, just don't list "good communicator" as a skill and instead talk about your communication skills when you talk about your experience. This is also true for a lot of soft skills such as "effective team member" or "strong personal leadership".
 
+Similarily, show that you know programming languages by mentioning them in the work you've done with them and don't just list a bunch of them at the top of your resume. It's useful to have them all in one place of course. But as a general rule of thumb, any language/technology/acronym that you put in a list should also appear elsewhere on your resume.
```

Comment:
I don't actually have the list anymore and it's been working okay for me

"It's useful" -> "It can be useful"

Also, sometimes you can have things in the list that you know but haven't used in recent interesting projects that made it onto your resume. In those cases I think it's fine to not have it appear somewhere else, but also know that it isn't making much of an impact (oh you talked about that a bit in the paragraph below, but that's a much more nerdy example - Haskell for fun challenge vs just having some smaller projects that don't matter as much anymore. It might be more inclusive to generalize that example a bit)